MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -30,7 +30,7 @@ import Foundation
     @objc public var account: MSALAccount
 
     let configuration: MSALNativeAuthConfiguration
```

Comment:
can this be internal too? Better to use the same access control for these two fields

Reply:
Done